spec/requests/flows/assertion_spec.rb

```diff
@@ -1,33 +1,78 @@
 require 'spec_helper_integration'
 
-feature 'Resource Owner Assertion Flow inproperly set up' do
-  background do
+describe 'Resource Owner Assertion Flow inproperly set up', type: :request do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.